src/Describer/DeleteAnnotationDescriber.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopping\ApiTKManipulationBundle\Describer;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Nelmio\ApiDocBundle\OpenApiPhp\Util;
+use Nelmio\ApiDocBundle\RouteDescriber\RouteDescriberInterface;
+use OpenApi\Annotations as OA;
+use ReflectionAttribute;
+use ReflectionMethod;
+use Shopping\ApiTKCommonBundle\Describer\RouteDescriberTrait;
+use Shopping\ApiTKManipulationBundle\Annotation\Delete;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Route;
+
+/**
+ * Provides automatic Parameter OpenApi annotations for actions that use the
+ * param converter for Delete annotation.
+ */
+class DeleteAnnotationDescriber implements RouteDescriberInterface
+{
+    use RouteDescriberTrait;
+
+    public function __construct(
+        private AnnotationReader $annotationReader
+    ) {
+    }
+
+    public function describe(OA\OpenApi $api, Route $route, ReflectionMethod $reflectionMethod): void
+    {
+        $deletes = $this->getAnnotations($reflectionMethod);
+        if (empty($deletes)) {
+            return;
+        }
+
+        foreach ($this->getOperations($api, $route) as $operation) {
+            $this->addDeletesToOperation($operation, $deletes, $route);
+        }
+    }
+
+    /**
+     * @param Delete[] $deletes
+     */
+    private function addDeletesToOperation(OA\Operation $operation, array $deletes, Route $route): void
+    {
+        $routePlaceholders = [];
+        preg_match_all('/{([^}]+)}/', $route->getPath(), $routePlaceholders);
+        $routePlaceholders = $routePlaceholders[1];
```

Comment:
I'd recommend to use `$routePlaceholders = $routePlaceholders[1] ?? [];` instead. Then your method won't break if you'd change the regexp.

```suggestion
        $routePlaceholders = $routePlaceholders[1] ?? [];
```

Reply:
Added in https://github.com/byWulf/apitk-manipulation-bundle/pull/12/commits/37555c8c71cc69fafa19cbfded1942048e6079e6